### ISSUE TITLE
Progress 0.9.9

### ISSUE THREAD

**Issue by realKSMC** [Aquerr/EagleFactions#46]
What about releasing the version 0.9.9?
Can't wait for it :)

**Comment by Aquerr** [comment]
Yes, I am actually testing it now so I should be available soon :)

**Comment by realKSMC** [comment]
Wow, sounds fantastic! :) So, today? ;)

**Comment by Aquerr** [comment]
Yeah! I hope so! 😄

**Comment by realKSMC** [comment]
Me too :)

**Comment by Aquerr** [comment]
I have tested everything now. It should be good... but because it is still beta so some bugs can show up (hope there are none). If you find something in the future, please tell me about it. 😄

New version should be available in about 1h or something like this.

**Comment by realKSMC** [comment]
Alright :) Ofc I will just open new issues :P
